CMakeLists.txt

```diff
@@ -195,7 +195,9 @@ option(IREE_BUILD_EXPERIMENTAL_WEB_SAMPLES "Builds experimental web samples." OF
 #-------------------------------------------------------------------------------
 
 set(IREE_CUDA_AVAILABLE OFF)
-find_package(CUDAToolkit)
+# The IREE cuda2 driver requires CUDA >= 12.
```

Comment:
Looks like this may fix https://github.com/openxla/iree/issues/14420 . I've context switched out some of the details there. Can you give that a read and see if anything is missing?

Reply:
I don't think we need the "install our own version if the user provided a version that is too old" from that... that was just an idea to make things a bit easier for some users.

Reply:
It is relevant. The solution in this PR is to
ignore the detected CUDA Toolkit and download our own version if the detected version is < 12.
This is one of the approaches you proposed there to handle an old CUDA version.

Reply:
Ah, I see. Thanks!
(plus sorry for the delay, I'm working through a backlog of emails etc.)